src/index.js

```diff
@@ -327,6 +386,16 @@ class GossipSub extends BasicPubsub {
     this.control = new Map()
   }
 
+  _subscribe (topics) {
```

Comment:
Any reason for not overriding `subscribe` and `unsubscribe`? Also, please add js docs with the @override annotation